docs/guide/performance.md

```diff
@@ -43,12 +43,18 @@ In production setups:
 - Keep Composer up to date.
 - Always dump optimized class map using `composer dump-autoload --optimize` or relevant flags on `composer install/update`.
 
-### Redis
-
-- Redis is recommended over Memcached as the latter has big performance issues caused by how Symfony cache works for cache tagging (Memcached does not handle lookups of non-existing cache items very well).
-- Redis is recommended over filesystem cache even with a single server, as it offers better general performance for operations invalidating cache.
-    - However, pure read performance is slower, especially if the next point is not optimized.
-- If you use Redis, make sure to tune it for in-memory cache usage. Its persistence feature is not needed with eZ Platform cache and will severely slow down execution time.
+### Memcached/Redis
```

Comment:
doc review note, this moves the header back to what it was, and afaik still is in  1.13 doc